app/src/main/java/com/beemdevelopment/aegis/ui/PreferencesFragment.java

```diff
@@ -102,6 +105,7 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
         AegisApplication app = (AegisApplication) getActivity().getApplication();
         _prefs = app.getPreferences();
         _vault = app.getVaultManager();
+        _slots = _vault.getCredentials().getSlots();
```

Comment:
This will become invalid if the user disables and then re-enables encryption. Can you place the ``getSlots()`` call in-line?